app/main.py

```diff
@@ -1,29 +1,29 @@
 class Person:
     people = {}
 
-    def __init__(self, name, age, wife=None, husband=None):
+    def __init__(self, name: str, age: int):
         self.name = name
         self.age = age
-        if wife is not None:
-            self.wife = wife
-        if husband is not None:
-            self.husband = husband
-        self.__class__.people[self.name] = self
+        self.people[self.name] = self
+
+    @classmethod
+    def is_wife(cls, person):
+        if "wife" in person and person["wife"] is not None:
+            if person["wife"] in cls.people:
+                cls.people[person["name"]].wife = cls.people[person["wife"]]
+                cls.people[person["wife"]].husband = cls.people[person["name"]]
+
+    @classmethod
+    def is_husband(cls, pers):
```

Comment:
i think better use the same name like in wife check
And use 
person: dict 
Good practice too